src/package.json

```diff
@@ -12,6 +12,7 @@
     "auto-launch": "^5.0.1",
     "bootstrap": "^3.3.7",
     "create-react-class": "^15.5.3",
+    "deepmerge": "^1.5.1",
```

Comment:
Is assume this is for https://github.com/KyleAMathews/deepmerge?

Reply:
correct, also seen here https://www.npmjs.com/package/deepmerge